### PR TITLE
Updated my profile name, readme

### LINKED ISSUE #650: Updating My Profile Name, and README.
I have updated my section, by changing my name and image url. thanks

### PR DESCRIPTION
**Pull request by CodingSpecies**
Updated My name, and readme. thanks
Resolve #650

### CODE REVIEW

**Comment by CodingSpecies** [comment]
#650

File: data.json [hunk 1]
```diff
@@ -1158,10 +1158,10 @@
     "issueNumber": 602
   },
   {
-    "name": "Noorin Sakhi",
-    "githubUsername": "NoorinSakhi",
-    "imageUrl": "https://user-images.githubusercontent.com/70807500/126067761-2fd771c0-0c5c-4748-ab0d-429250027830.jpg",
-    "issueNumber": 604
+    "name": "CodingSpecies",
```
Comment (Panquesito7) [inline data.json:1161]: I think you should keep `Noorin Sakhi` here if it's your IRL name (if you want).

File: data.json [hunk 2]
```diff
@@ -1158,10 +1158,10 @@
...
+    "githubUsername": "CodingSpecies",
+    "imageUrl": "https://user-images.githubusercontent.com/70807500/129594715-ef05ecc0-720f-4b75-a507-c2afed68cf02.png",
```
Comment (Panquesito7) [inline data.json:1163]: I suggest more of an image like this.

image

```suggestion
    "imageUrl": "https://user-images.githubusercontent.com/51391473/129598330-ca71253a-4431-4bbd-91af-8db083144b9a.png",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: data.json (hunk 1 above)
Comment (CodingSpecies) [reply data.json:1161]: I think I am going to leave that, but I changed the image. thank you @panquesito7

**Review by Panquesito7 — APPROVED** [review]
LGTM. Thank you for your contributions, @CodingSpecies! 😄👍🎉
@schmelto, can be merged once the issue is linked with this PR.

**Review by schmelto — APPROVED** [review]

**Comment by schmelto** [comment]
> LGTM. Thank you for your contributions, @CodingSpecies! 😄👍🎉
> @schmelto, can be merged once the issue is linked with this PR.
> 

Yes thanks I've added resolve #issuenumber to the description

**Comment by schmelto** [comment]
> #650 

By adding resolve #yourissuenumber to the pr description it will automatically close the issue on pr merge I've added this one for you but you can keep it in mind for future PRs

**Comment by CodingSpecies** [comment]
> > #650
> 
> By adding resolve #yourissuenumber to the pr description it will automatically close the issue on pr merge I've added this one for you but you can keep it in mind for future PRs

ok, thank you @schmelto

**Pull request merged**